packages/server/src/workers/utils.ts

```diff
@@ -114,20 +114,25 @@ export async function isFhirCriteriaMet(subscription: Subscription, currentResou
   if (!criteria?.valueString) {
     return true;
   }
-  const history = await systemRepo.readHistory(currentResource.resourceType, currentResource?.id as string);
+  const previous = await getPreviousResource(currentResource);
   const evalInput = {
     '%current': toTypedValue(currentResource),
-    '%previous': toTypedValue({}),
+    '%previous': toTypedValue(previous ?? {}),
```

Comment:
I'm curious if something this might work better:
```suggestion
    '%previous': previous ? toTypedValue(previous) : undefined,
```

Reply:
We may want to address that change in https://github.com/medplum/medplum/pull/3808 but FWIW, the `variables` typing as input to `evalFhirPathTyped` would not allow a value of `undefined`. It must be a `TypedValue`.